rockcraft.yaml

```diff
@@ -25,6 +25,21 @@ services:
       SPARK_USER_DATA: /home/spark
     on-success: shutdown
     on-failure: shutdown
+  history-server:
```

Comment:
I am a bit confused about this.

Here are my assumptions (you can ignore the comment if they are wrong):
 - you are aiming to have **a single image** 
 - the outcome ROCK from Rockcraft is actually never published and only used for the Dockerfile 

If this is true, then you'd never use this Pebble service cause your image's entrypoint has the `entrypoint` service hardcoded.

Reply:
Yes, but for instance in charms, the entrypoint is overridden with Pebble only. In that case, it could be good to already have a pebble layer, that can be customized by the charm. Also when running the image using normal Pebble syntax one could still enable the history-server using:

```docker run charmed-spark \; start history-server```